### ISSUE TITLE
Map-placed random arcades do not have orientation when the map generates

### ISSUE THREAD

**Issue by Helianthus00** [tgstation/tgstation#53479]

**Comment by Jared-Fogle** [comment]
Am I misunderstanding something?


![2020-09-06T00-14-09](https://user-images.githubusercontent.com/35135081/92320391-f83c5f00-efd5-11ea-971f-58a8f2c9c380.gif)

**Comment by Helianthus00** [comment]
Apologies for the confusing wording on my part. I meant that the map obj did not spawn the correctly rotated machine. Anyway, I tested it for pictures, and I'm glad I did, since it seems only the random ones are broken. I went ahead and tested all of the machines and subtypes and included a labeled image with the results. 
![arcademap](https://user-images.githubusercontent.com/64755361/92320721-bfee4e00-efe0-11ea-86fc-86035ab69849.PNG)
![arcade](https://user-images.githubusercontent.com/64755361/92320722-c11f7b00-efe0-11ea-811d-5a8a6962d660.PNG)
